docs/docs/legal/data-processing-agreement.mdx

```diff
@@ -1,113 +1,276 @@
 ---
 title: Data Processing Agreement
 custom_edit_url: null
-custom:
-    created_at: 2022-07-15
-    updated_at: 2023-11-16
 --- 
-import PiidTable from './_piid-table.mdx';
 
-Last updated on November 15, 2023
+Last updated on  20 March, 2025
 
-Within the scope of the [**Framework Agreement**](terms-of-service), the **Processor** (CAOS Ltd., also **ZITADEL**) processes **Personal Data** on behalf of the **Customer** (Responsible Party), collectively the **"Parties"**.
+This Data Protection Agreement and its annexes (“**DPA**”) forms part of the [Framework Agreement](./terms-of-service) between ZITADEL and the Customer in respect of the provision of certain services, including any applicable statement of work, booking, purchase order, PO or any agreed upon instructions (the "**Agreement**") and applies where, and to the extent that, ZITADEL processes Personal Data as a Processor on behalf of the Customer under the Framework Agreement (each a “**Party**” and together the “**Parties**”).
 
-This Annex to the Agreement governs the Parties' data protection obligations in addition to the provisions of the Agreement.
+All capitalized terms not defined in this DPA will have the meanings set forth in the Agreement.
+Any privacy or data protection related clauses or agreement previously entered into by ZITADEL and the Customer, with regards to the subject matter of this DPA, will be superseded by and replaced with this DPA.
+No one other than a Party to this DPA, their successors and permitted assignees will have any right to enforce any of its terms.
 
-## Subject matter, duration, nature and purpose of the processing as well as the type of personal data and categories of data subjects
+This Addendum shall become legally binding upon Customer entering into the Agreement.
```

Comment:
Is Addendum meant as "Addendum" to the Framework Agreement?

Reply:
Needs to be DPA